indico/web/templates/_sortable_list.html

```diff
@@ -0,0 +1,55 @@
+{% macro sortable_items(items, actions_macro, draggable, toggle_enabled) -%}
```

Comment:
forgot to delete the other template? if not: see above regarding using caller instead of passing around macros